flutter/lib/src/sentry_flutter.dart

```diff
@@ -48,6 +48,9 @@ mixin SentryFlutter {
   }) async {
     final flutterOptions = SentryFlutterOptions();
 
+    // ignore: invalid_use_of_internal_member
+    mainIsolateStartTime = flutterOptions.clock();
```

Comment:
this should be removed, same reaason of the other PR (not changing the mainIsolateStartTime after being initialized)

Reply:
just leaving a comment for future readers. Dart static variables are lazy, initialized only when accessed, so we cannot rely on static variables.